tests/src/tests/try.spec.ts

```diff
@@ -181,4 +181,52 @@ export = () => {
 
 		expect(foo()).to.equal(2);
 	});
+
+	it("should rethrow the error if there's no catch block", () => {
+		function foo() {
+			try {
+				throw "bar";
+			} finally {
+			}
+		}
+
+		expect(() => foo()).to.throw("bar");
+	});
+
+	it("should run try -> catch -> finally in order", () => {
+		const array = new Array<number>();
+		try {
+			let condition = true;
```

Comment:
This `let condition` is to prevent TypeScript from complaining about `array.push(999);` being unreachable code.

Reply:
This would generate an eslint warning for prefer-const. I would suggest doing `true as boolean` rather than the `let`.